packages/connect/src/data/firmwareInfo.ts

```diff
@@ -480,6 +475,10 @@ export const getReleaseInfo = ({
     const { rollout_probability } = conditions;
     const shouldBeOffered = calculateShouldOfferRelease(rollout_probability, features.device_id);
 
+    if (requiresIntermediary && intermediary) {
+        isNewer = true;
+    }
```

Comment:
To be sure: This means that whenever an intermediary firmware is needed to install, it is signalled that there's update available?

Reply:
Yes.